CorinneTest/IPersonFactory.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
```

Comment:
are you actually using these interfaces?

Reply:
no - deleting them.